### ISSUE TITLE
Permission Denied

### ISSUE THREAD

**Issue by AyYildizPilot** [codebox/mazes#1]
Hello,
Your codes don't download normally like everyone else's?
I tried on windows and ubuntu systems and failed.
The error message below is the yes command doesn't work. I found the document that needs to be done on the Ubuntu system, but now the error is different from the following error, it gives an access denied warning and I can't do anything.

what is it called subdirectories? is it possible to add them all here?

The authenticity of host 'github.com (140.82.121.4)' can't be established.
ED25519 key fingerprint is SHA256:+DiY3wxxxxxF/zLDA0z.........qU.
This key is not known by any other names.
Are you sure you want to continue connecting (yes/no/[fingerprint])?

**Comment by codebox** [comment]
I don't think this is anything I can help with, it sounds like a local git configuration issue. The first time you pull a project from github you will need to answer 'yes' to the question above, if you have problems after that I don't know what is causing them. There are no restrictions on cloning the repo, it's a standard 'public' one that everyone has permission to use.

**Comment by AyYildizPilot** [comment]
Thanks for the answer,

The problem is related to SSH sharing, as you mentioned, it is within the scope of github and ssh settings.

When I try the other https method (git clone https://github.com/codebox/mazes.git) the files are copied.
When I start the project and log in to the https://0.0.0.0:8000 url, only the right menus appear and the page does not work.

Is there an external solution to ssh clone?
Do you need to pull files from different sources while cloning, is it not possible to share them in a single package from the Releases section?

**Comment by codebox** [comment]
Did you use the git command from the readme, that should pull down everything you need: https://github.com/codebox/mazes#running-locally

**Comment by AyYildizPilot** [comment]
The source of the error I get the first message is the command there.

The ssh command gives an error, and there is no problem because other people on github share their projects with https.

`git clone --recurse-submodules git@github.com:codebox/mazes.git`

```
The authenticity of host 'github.com (140.82.121.4)' can't be established.
ED25519 key fingerprint is SHA256:+DiY3wvv....................................
This key is not known by any other names.
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Could not create directory '/z/.ssh' (Permission denied).
Failed to add the host to the list of known hosts (/z/.ssh/known_hosts).
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```

**Comment by codebox** [comment]
Isn't this the problem? 
`Could not create directory '/z/.ssh' (Permission denied).`
Looks like it thinks `/z` is your home directory and it can't create a .ssh directory there

**Comment by AyYildizPilot** [comment]
Very complex subject, google searches tired. Everyone has suggested something, but there is no solution. Today I'm trying with Windows 10 git-scm portable version, z/ error may be caused by it.
SSH key continued in Ubuntu 23.04 version. The Ubuntu system computer is at home, I'll try again when I get home.

I solved the problem with manual installation on Windows system.

I cloned first, subdirectory does not come with this command.
`git clone https://github.com/codebox/mazes.git`

I searched where the subdirectory was in js/lib.

`cd mazes/js/lib`

i cloned subdirectory
`git clone https://github.com/codebox/maze.js.git`
I manually moved the files in the maze.js/ folder to the parent directory (mazes/js/lib/)

`python3 -m http.server`

I started with, when I checked http://127.0.0.1:8000 I came across maze it works.
Those who have problems like me can provide a solution this way.
